src/errors.md

```diff
@@ -0,0 +1,40 @@
+# Common errors
+
+## Errors
+
+### You do not have access to this instance
+
+<img src="../../_images/no_access_instance.png" alt="No Access" width="1300" center/>
+
+- Why:
```

Comment:
Unsure if that many bullet points look good, I would probably use headers for why, origins and remediations